integration-cli/docker_cli_events_test.go

```diff
@@ -206,18 +206,26 @@ func TestEventsImagePull(t *testing.T) {
 func TestEventsImageImport(t *testing.T) {
 	since := time.Now().Unix()
 
-	defer deleteImages("cirros")
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "true")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		t.Fatal("failed to create a container", out, err)
+	}
+	cleanedContainerID := stripTrailingCharacters(out)
+	defer deleteContainer(cleanedContainerID)
 
-	server, err := fileServer(map[string]string{
-		"/cirros.tar.gz": "/cirros.tar.gz",
-	})
+	exportCmd := exec.Command(dockerBinary, "export", cleanedContainerID)
+	importCmd := exec.Command(dockerBinary, "import", "-")
+	importCmd.Stdin, err = exportCmd.StdoutPipe()
 	if err != nil {
-		t.Fatal(err)
+		t.Errorf("export StdoutPipe failed with errors: %v", err)
+	}
+	err = exportCmd.Start()
+	if err != nil {
+		t.Errorf("export failed with errors: %v", err)
 	}
-	defer server.Close()
-	fileURL := fmt.Sprintf("%s/cirros.tar.gz", server.URL)
-	importCmd := exec.Command(dockerBinary, "import", fileURL, "cirros")
-	out, _, err := runCommandWithOutput(importCmd)
+	out, _, err = runCommandWithOutput(importCmd)
+	exportCmd.Wait() // release "exportCmd" resources
```

Comment:
actually I wondered if we need this at all. if the other end of pipe is broken, this cmd will also exit right away, correct?